walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,116 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
+
+  // Takes in a collection of all events (each with seperate attendees and timeranges) and
+  // 'MeetingRequest' (with required attendees and a duration requirement) and returns a Collection
+  // of TimeRanges that work with schedules of required attendees and meets the duration requirement
```

Comment:
Prefer to use javadocs over comments when writing documentation for public classes and methods:
https://google.github.io/styleguide/javaguide.html#s7-javadoc

The same goes for other public methods in this class

In addition to this recommendation in the style guide, if you're using an IDE or a program for browsing source code, there will usually be functionality which displays the javadoc for a method when you hover over it at one of its call sites. The same isn't true for comments, so this makes javadocs a lot more useful for public methods.

Reply:
Still a little unsure how to use Javadocs but I think I get it. Wasn't sure if my "summary" should be in header tags or not.